lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -339,9 +339,10 @@ func listEnodebs(c echo.Context) error {
 	for _, ent := range ents {
 		enodebs[ent.Key] = (&lte_models.Enodeb{}).FromBackendModels(ent)
 	}
+	token := lte_models.PageToken(nextPageToken)
 	paginatedEnodebs := &lte_models.PaginatedEnodebs{
 		Enodebs:    enodebs,
-		PageToken:  lte_models.PageToken(nextPageToken),
+		PageToken:  &token,
```

Comment:
For my understanding: we can't do this inline/single-line?

Reply:
Yes, the address operator `&` requires an operand that is addressable which is not given inline. It would result in `invalid composite literal type`.